internal/perf/cpu.go

```diff
@@ -44,7 +43,10 @@ func HandleCPUProfileSignals() {
 		}()
 
 		// Profile.
-		pprof.StartCPUProfile(f)
+		err = pprof.StartCPUProfile(f)
+		if err != nil {
+			logger.Infof("StartCPUProfile failed: %v", err)
```

Comment:
can we make it a warning or error log.

Reply:
also does it make sense to return from here instead of executing further steps?

Reply:
func StartCPUProfile returns an error if profiling is already enabled. So, I think we should not break the flow here. I've changed the log level to ERROR.

Reply:
how does it make sense to run these 2 steps if profile start itself failed:
time.Sleep(duration)
pprof.StopCPUProfile()